packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -814,6 +808,11 @@ export class GalleryContainer extends React.Component {
             <style dangerouslySetInnerHTML={{ __html: this.dynamicStyles }} />
           )}
         </div>
+        {this.props.proGalleryRole === 'application' && (
+          <span ref={(e) => this.outOfViewComponent = e} tabIndex={-1} className="sr-only">
+            out of gallery
```

Comment:
Need to check what to do about this plain text here

Reply:
we will add a key

Reply:
Also please change the className (something like "sr-only-XXXXX") or 2 class names, such as "sr-only XXXXXX"